draft-ietf-quic-transport.md

```diff
@@ -1483,10 +1483,13 @@ subsequent sections.
 ## STREAM Frame {#frame-stream}
 
 STREAM frames implicitly create a stream and carry stream data. The type byte
-for a STREAM frame contains embedded flags, and is formatted as `11FSSOOD`.
+for a STREAM frame contains embedded flags, and is formatted as `1UFSSOOD`.
 These bits are parsed as follows:
 
-* The first two bits must be set to 11, indicating that this is a STREAM frame.
+* The first bit must be set to 1, indicating that this is a STREAM frame.
+
+* 'U' is the UNI bit, which is used to indicate this is a unidirectional
+  stream, and no stream frames may be sent in response.
```

Comment:
@kazuho Ryan's reasoning is the reason I didn't go with that approach, along with the fact that even though determining if a number is even or odd is easy, determining if it's mod 4 value is 1 or 3 is not as simple, so I thought it would increase the complexity of debugging.